tasks/update-granules-cmr-metadata-file-links/package.json

```diff
@@ -49,7 +49,6 @@
     "lodash": "^4.17.15"
   },
   "devDependencies": {
-    "@cumulus/aws-client": "20.0.0",
```

Comment:
This entry is already in dependencies and causes lerna not be able to update version https://github.com/nasa/cumulus/pull/3925#discussion_r1980170454